src/Generator.Tests/premake5.lua

```diff
@@ -6,12 +6,17 @@ project "CppSharp.Generator.Tests"
   files { "**.cs" }
   vpaths { ["*"] = "*" }
 
-  libdirs 
+  libdirs
```

Comment:
I don't know why that happened but i can restore the empty space if you want. I really don't know if it is necessary or not.